src/vcastDataServer.ts

```diff
@@ -143,6 +143,7 @@ async function startServer() {
     cwd: serverCWD,
   });
 
+  await new Promise((resolve) => setTimeout(resolve, 5000));
```

Comment:
Hmm, what's this about?

Reply:
Deleted in 3509429a2b77635f18359b6f01833aa8409d9dd9.